sdk/communication/test-resources.json

```diff
@@ -13,11 +13,37 @@
             "defaultValue": "communication",
             "type": "string"
         },
+        "communicationServicesEndpointSuffix": {
+            "defaultValue": ".communication.azure.com",
+            "type": "string"
+        },
+        "subscriptionId": {
+            "defaultValue": "[subscription().subscriptionId]",
+            "type": "string"
+        },
         "testApplicationOid": {
             "type": "string",
             "metadata": {
                 "description": "The client OID to grant access to test resources."
             }
+        },
+        "tenantId": {
+            "type": "String",
```

Comment:
String should be lowercase, there's a couple spots where this needs to be done

Reply:
Done!